runelite-client/src/main/java/net/runelite/client/plugins/deathindicator/DeathIndicatorConfig.java

```diff
@@ -0,0 +1,236 @@
+/*
+ * Copyright (c) 2017, Adam <Adam@sigterm.info>
```

Comment:
Is this copyright correct? New files only need your own name unless you mean to credit it to the RuneLite project specifically.

Reply:
It's not, I'll go ahead and remove adam's name from the copyright :P (Copypaste strikes again).